docs/INSTALL.md

```diff
@@ -86,67 +79,73 @@ Here is the list of DC/OS versions supported on dcos-terraform natively:
+The default variables inputs are tracked in the [terraform-azurerm-dcos](https://registry.terraform.io/modules/dcos-terraform/dcos/azurerm) terraform registry.
 
-For advanced users with stringent requirements, here are DC/OS flag examples you can simply paste in `cluster_profile.tfvars`.
 
 ```bash
-$ cat cluster_profile.tfvars
-dcos_version = "1.11.1"
-os = "centos_7.3"
-num_masters = "3"
-num_private_agents = "2"
-num_public_agents = "1"
-ssh_key_name = "default" 
-dcos_cluster_name = "DC/OS Cluster"
-dcos_cluster_docker_credentials_enabled =  "true"
-dcos_cluster_docker_credentials_write_to_etc = "true"
-dcos_cluster_docker_credentials_dcos_owned = "false"
-dcos_cluster_docker_registry_url = "https://index.docker.io"
-dcos_use_proxy = "yes"
-dcos_http_proxy = "example.com"
-dcos_https_proxy = "example.com"
-dcos_no_proxy = <<EOF
-# YAML
- - "internal.net"
- - "169.254.169.254"
-EOF
-dcos_overlay_network = <<EOF
-# YAML
-    vtep_subnet: 44.128.0.0/20
-    vtep_mac_oui: 70:B3:D5:00:00:00
-    overlays:
-      - name: dcos
-        subnet: 12.0.0.0/8
-        prefix: 26
-EOF
-dcos_rexray_config = <<EOF
-# YAML
-  rexray:
-    loglevel: warn
-    modules:
-      default-admin:
-        host: tcp://127.0.0.1:61003
-    storageDrivers:
-    - ec2
-    volume:
-      unmount:
-        ignoreusedcount: true
-EOF
-dcos_cluster_docker_credentials = <<EOF
-# YAML
-  auths:
-    'https://index.docker.io/v1/':
-      auth: Ze9ja2VyY3licmljSmVFOEJrcTY2eTV1WHhnSkVuVndjVEE=
-EOF
+$ cat main.tf
+...
+module "dcos" {
+  source = "dcos-terraform/dcos/azurerm"
+
+  # additional example variables in the module
+  dcos_version = "1.11.5"
+  dcos_instance_os = "centos_7.3"
+  num_masters = "3"
+  num_private_agents = "2"
+  num_public_agents = "1"
+  dcos_cluster_name = "DC/OS Cluster"
+  dcos_cluster_docker_credentials_enabled =  "true"
+  dcos_cluster_docker_credentials_write_to_etc = "true"
+  dcos_cluster_docker_credentials_dcos_owned = "false"
+  dcos_cluster_docker_registry_url = "https://index.docker.io"
+  dcos_use_proxy = "yes"
+  dcos_http_proxy = "example.com"
+  dcos_https_proxy = "example.com"
+  dcos_no_proxy = <<EOF
+  # YAML
+   - "internal.net"
+   - "169.254.169.254"
+  EOF
+  dcos_overlay_network = <<EOF
+  # YAML
+      vtep_subnet: 44.128.0.0/20
+      vtep_mac_oui: 70:B3:D5:00:00:00
+      overlays:
+        - name: dcos
+          subnet: 12.0.0.0/8
+          prefix: 26
+  EOF
+  dcos_rexray_config = <<EOF
+  # YAML
+    rexray:
+      loglevel: warn
+      modules:
+        default-admin:
+          host: tcp://127.0.0.1:61003
+      storageDrivers:
+      - ec2
+      volume:
+        unmount:
+          ignoreusedcount: true
+  EOF
+  dcos_cluster_docker_credentials = <<EOF
+  # YAML
+    auths:
+      'https://index.docker.io/v1/':
+        auth: Ze9ja2VyY3licmljSmVFOEJrcTY2eTV1WHhnSkVuVndjVEE=
+  EOF
 ```
 **Note**: The YAML comment is required for the DC/OS specific YAML settings.
```

Comment:
<p class="message--note"><strong>NOTE: </strong>The YAML comment is required for the DC/OS specific YAML settings.</p>

Reply:
Remove the content in line #150 and copy-paste the above line.

Reply:
Done